gui/src/app/view/settings.rs

```diff
@@ -244,18 +249,59 @@ pub fn bitcoind_edit<'a>(
 
     col = col
         .push(
-            Column::new()
-                .push(text("Cookie file path:").bold().small())
+            [RpcAuthType::CookieFile, RpcAuthType::UserPass]
+                .iter()
+                .fold(
+                    Row::new()
+                        .push(text("RPC authentication:").small().bold())
+                        .spacing(10),
+                    |row, auth_type| {
+                        row.push(radio(
+                            format!("{}", auth_type),
+                            *auth_type,
+                            Some(*selected_auth_type),
+                            SettingsEditMessage::BitcoindRpcAuthTypeSelected,
+                        ))
+                        .spacing(30)
+                        .align_items(Alignment::Center)
```

Comment:
Nice, i like the radio button. Just let's not make them green?

Reply:
Looking through the rest of the interface most buttons do get green when hovered though.